.codeclimate.yml

```diff
@@ -0,0 +1,21 @@
+# Save as .codeclimate.yml (note leading .) in project root directory
```

Comment:
This file is new to me. I'll have to look this up.

Reply:
It tracks style issues:  https://codeclimate.com/github/pysat/pysatMissions